plugins/controlboard/src/ControlBoardDriverCoupling.cpp

```diff
@@ -564,3 +565,239 @@ yarp::sig::Vector CerHandCouplingHandler::decoupleRefTrq (yarp::sig::Vector& trq
     out[m_coupledJoints[3]] = trq_ref[m_coupledJoints[1]]/2;
     return out;
 }
+
+//------------------------------------------------------------------------------------------------------------------
+// HandMk3CouplingHandler
+//------------------------------------------------------------------------------------------------------------------
+
+HandMk3CouplingHandler::HandMk3CouplingHandler(gazebo::physics::Model* model, yarp::sig::VectorOf<int> coupled_joints, std::vector<std::string> coupled_joint_names)
+: BaseCouplingHandler(model, coupled_joints,coupled_joint_names), LUTSIZE(4096)
+{
+    const double RAD2DEG = 180.0/atan2(0.0,-1.0);
+    const double DEG2RAD = 1.0/RAD2DEG;
+    
+    m_couplingSize = 11;
+    
+    thumb_lut = new double[LUTSIZE];
+    index_lut = new double[LUTSIZE];
+    
+    double num[LUTSIZE];
```

Comment:
Unfortunately Variable Length Arrays (VLA) are not part of the C++ standard (see https://stackoverflow.com/questions/1887097/why-arent-variable-length-arrays-part-of-the-c-standard), so to avoid problems in the future it is better to avoid their use. As this function is not called at every simulation step, I think it is ok to just use a `std::vector` here as well: 
```suggestion
    std::vector<double> num(LUTSIZE);
```